app/controllers/notifications_controller.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# routes
+class NotificationsController < ApplicationController
+  def index
+    @notifications = Notification.all_unviewed(current_user).latest
+
+    respond_to do |format|
+      format.html
+      format.js
+    end
+  end
+
+  def update
+    @notification = Notification.find_id(params[:id])
+    message =
+      if @notification.update(notification_params)
```

Comment:
refactor to a single line and use a ternary operator

Reply:
Done!

Reply:
Done!